src/components/ExportModal.vue

```diff
@@ -0,0 +1,113 @@
+<script setup lang="ts">
+interface Props {
+  buttonText?: string;
+  dialogTitle?: string;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  buttonText: 'Open Text Area',
+  dialogTitle: 'Text Area Dialog',
+});
+
+const dialogOpen = ref(false);
+const textContent = ref('');
+const toast = useToast();
+
+function openDialog() {
+  // Get ftAuthLoginInfo from localStorage and convert to base64
+  const authInfo = localStorage.getItem('ftAuthLoginInfo');
+  if (authInfo) {
+    try {
+      // Parse the authInfo JSON
+      const parsedAuthInfo = JSON.parse(authInfo);
+
+      // Extract the required attributes from each bot entry
+      const extractedBots = Object.values(parsedAuthInfo).map((bot: any) => ({
+        botName: bot.botName,
+        url: bot.apiUrl,
+        username: bot.username,
+        accessToken: bot.accessToken,
+        refreshToken: bot.refreshToken,
```

Comment:
As mentioned in the other comment - i think from a security standpoint, tokens shouldn't be exported.

you'll still be able to migrate most info - and simply have to provide your password for each bot again. 
Essentially - it's the same process as what happens about once a month - when the refresh-token expires.

(please also note and fix the CI errors - they're obviously blocking without anyone mentioning it explicitly) ;)